app/assets/javascripts/components/header.es6.jsx

```diff
@@ -3,14 +3,26 @@ class Header extends React.Component {
   get styles() {
     return {
       container: {
+        // display: flex,
         position: 'relative',
         width: '100%',
         height: 48,
-        backgroundColor: '#68B1DE',
+        backgroundColor: '#fff',
+        padding: 10,
+        boxShadow: '0 0 25px 0 rgba(0, 0, 0, 0.15)',
       },
       title: {
         position: 'relative',
-        color: '#ffffff',
+        verticalAlign: 'middle',
+        color: '#68B1DE',
+        fontSize: 25,
```

Comment:
In general, font size should have `px` because the units matter. For padding if its just one number you don't need to specify `px` and can just leave it as a number. There's a few different cases that you'll see in the future and we'll talk more about it then.